### ISSUE TITLE
Suggestion on the ultimate set of rules for a security investigation

### ISSUE THREAD

**Issue by jonny-wg2** [Issif/falco-talon#7]
Cool project. A couple of thoughts I have on how I could find this useful.

In the scenario of a compromised pod (e.g. bitcoin mining identified)

1. dump metadata of the container for useful security investigation. E.g. processes, logs
2. add new labels and alert that a security investigation as begun in slack
3. create a netowork policy to restrict egress traffic (prevent remote access control) to isolate the pod

**Comment by Issif** [comment]
Hello,

Thank you for your message.

Your ideas are really valuable, and of course, more actions will be added to talon. This is a POC, I'm not currently convicted about how I managed the code, I'll change a lot soon for allowing people to add action easily, like with a framework.

For Slack notification, this is something I already started to work on:
![image](https://user-images.githubusercontent.com/4520100/147920247-829a9a18-2ac2-4957-a85c-3f24aa011d8b.png)
